static/styles/zulip.css

```diff
@@ -1437,6 +1444,12 @@ div.focused_table {
     display: block;
     position: relative;
     overflow: hidden;
+    /* We are using very high value here to imitate an infinite value so that
+    we can enable collapse/expand animation. */
+    max-height: 1000vh;
+    transition: max-height 0.5s ease-in-out, mask-size 0.3s ease-in-out;
```

Comment:
I don't think we want a 0.5s animation here; probably we don't want any at all (especially for closing the widget), but we generally shy away from relatively long animations that delay letting users see content that they asked us to display, since they're annoying.